spec/spec.md

```diff
@@ -540,6 +541,78 @@ TODO: Mention how this could relate to other specs such as Universal Wallet or W
 
 Example:
 
+```json
+{
+	"jobId": "1234",
+	"didState": {
+		"state": "action",
+		"action": "signPayload",
+		"signingRequest": {
+			"signingRequest1": {
+				"payload": {
+					...
+				},
+				"serializedPayload": "<-multibase->",
+				"kid": null,
+				"alg": "EdDsa",
+				"verificationMethod": "..." // could point to a verificationMethod, incl. VerifiableConditions with threshold, etc.
+				"proofPurpose": ".." // describes the purpose of the requested signature/proof
+			},
+			"signingRequest2": {
+				"serializedPayload": "<-multibase->",
+				"kid": null,
+				"alg": "ES256K"
+			}
+		}
+	},
+	"didRegistrationMetadata": {},
+	"didDocumentMetadata": {}
+}
+```
+
+#### `didState.action="decryptPayload"`
+
+This action indicates that the client needs to decrypt a payload, before the DID operation can be
+continued.
+
+This action is used in [Client Managed Secret Mode](#client-managed-secret-mode).
+
+With this action, the [`didState` output field](#didstate) MUST contain properties that indicate the payload
+to be decrypted, as well as additional information such as a key identifier or the decryption algorithm to be used.
+
+The [`didState` output field](#didstate) MAY contain additional properties that are relevant to this action.
+
+TODO: Need to specify in more detail how a DID Registrar requests decrypting, and how the client responds.
```

Comment:
This depends on the encrypted payload.
- access token (bearer?)
- signing request
- other?